### ISSUE TITLE
[Feat]: ability to export snapshots in the background

### ISSUE THREAD

**Issue by andrewm4894** [netdata/netdata-cloud#737]
### Problem

I have been stuck staring at this screen for about 5 mins or so while i wait on the snapshot

![image](https://user-images.githubusercontent.com/2178292/220884243-852e86b1-d3ad-492d-a77c-4b39dfead155.png)

### Description

It would be really nice if the snapshot could happen in the background in some way and i could just get it when its ready

### Importance

nice to have

### Value proposition

1. make snapshots easier to work with

**Comment by hugovalente-pm** [comment]
this feature is not exposed on new UI
